packages/dataviews/src/components/dataviews-pagination/index.tsx

```diff
@@ -37,41 +58,25 @@ function DataViewsPagination() {
 				<HStack
 					justify="flex-start"
 					expanded={ false }
-					spacing={ 2 }
-					className="dataviews-pagination__page-selection"
+					spacing={ 1 }
+					className="dataviews-pagination__page-select"
 				>
-					{ createInterpolateElement(
-						sprintf(
-							// translators: %s: Total number of pages.
-							_x( 'Page <CurrentPageControl /> of %s', 'paging' ),
-							totalPages
-						),
-						{
-							CurrentPageControl: (
-								<SelectControl
-									aria-label={ __( 'Current page' ) }
-									value={ view.page?.toString() }
-									options={ Array.from(
-										Array( totalPages )
-									).map( ( _, i ) => {
-										const page = i + 1;
-										return {
-											value: page.toString(),
-											label: page.toString(),
-										};
-									} ) }
-									onChange={ ( newValue ) => {
-										onChangeView( {
-											...view,
-											page: +newValue,
-										} );
-									} }
-									size="compact"
-									__nextHasNoMarginBottom
-								/>
-							),
-						}
-					) }
+					<div aria-hidden>{ __( 'Page' ) }</div>
+					<SelectControl
+						aria-label={ __( 'Current page' ) }
+						value={ currentPage.toString() }
+						options={ pageSelectOptions }
+						onChange={ ( newValue ) => {
+							onChangeView( {
+								...view,
+								page: +newValue,
+							} );
+						} }
+						size="compact"
+						__nextHasNoMarginBottom
+						variant="minimal"
+					/>
+					<div aria-hidden>/ { totalPages }</div>
```

Comment:
```suggestion
					<div aria-hidden>
						{ sprintf(
							// translators: Ttotal number of pages
							__( '/ %s' ),
							totalPages
						) }
					</div>
```

Would it make sense to make this translatable?

Reply:
Applied @jameskoster's suggestion in ac39701